playbooks/roles/xqueue/defaults/main.yml

```diff
@@ -62,6 +62,11 @@ XQUEUE_NEWRELIC_APPNAME: "{{ COMMON_ENVIRONMENT }}-{{ COMMON_DEPLOYMENT }}-xqueu
 XQUEUE_WORKERS: !!null
 XQUEUE_WORKERS_PER_QUEUE: 12
 
+XQUEUE_SESSION_ENGINE: 'django.contrib.sessions.backends.db'
+XQUEUE_CACHES:
+  default:
+    BACKEND: 'django.core.cache.backends.locmem.LocMemCache'
```

Comment:
Can't we use `!!null` here to have these keys excluded from the JSON file?  I can never remember the exact semantics of this, but it would be nice if we didn't need to copy the defaults from Django.

Reply:
Ah yeah, the xqueue settings code defaults to them already, so this is a good idea to leave it `!!null` here (I'm not sure of the syntax myself, but that sounds right).